// strmn/dist/index.js
function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var React = _interopDefault(require('react'));

var styles = {"test":"_3ybTi","buttonWrapper":"_3A3ND","primary":"_3ljpl","default":"_3UK6o","dashed":"_27xYC","text":"_5bXm4","link":"_71S8l"};

var ExampleComponent = function ExampleComponent(_ref) {
  var text = _ref.text;
  return /*#__PURE__*/React.createElement("div", {
    className: styles.test
  }, "Example Component: ", text);
};
var PrimaryButton = function PrimaryButton(_ref2) {
  var text = _ref2.text;
  return /*#__PURE__*/React.createElement("div", null, /*#__PURE__*/React.createElement("button", {
    className: styles.primary
  }, text));
};
var DefalultButton = function DefalultButton(_ref3) {
  var text = _ref3.text;
  return /*#__PURE__*/React.createElement("div", {
    className: styles.buttonWrapper
  }, /*#__PURE__*/React.createElement("button", {
    className: styles["default"]
  }, text));
};
var DashedButton = function DashedButton(_ref4) {
  var text = _ref4.text;
  return /*#__PURE__*/React.createElement("div", {
    className: styles.buttonWrapper
  }, /*#__PURE__*/React.createElement("button", {
    className: styles.dashed
  }, text));
};
var TextButton = function TextButton(_ref5) {
  var text = _ref5.text;
  return /*#__PURE__*/React.createElement("div", {
    className: styles.buttonWrapper
  }, /*#__PURE__*/React.createElement("button", {
    className: styles.text
  }, text));
};
var LinkButton = function LinkButton(_ref6) {
  var text = _ref6.text;
  return /*#__PURE__*/React.createElement("div", {
    className: styles.buttonWrapper
  }, /*#__PURE__*/React.createElement("button", {
    className: styles.link
  }, " ", text));
};

exports.DashedButton = DashedButton;
exports.DefalultButton = DefalultButton;
exports.ExampleComponent = ExampleComponent;
exports.LinkButton = LinkButton;
exports.PrimaryButton = PrimaryButton;
exports.TextButton = TextButton;
